docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,19 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service DCP Rollback to Zero (CB90067)
+
+*Background*: A service has been forced to DCP rollback to zero.
```

Comment:
Is there anything more we can say here about what a DCP rollback does? It's a relatively technical term that only people familiar with CB's internals will recognise.

Reply:
I was worried about putting in a more informative description because it may be too big. This is probably as consice as I can get it, how does it sound?

A DCP rollback is when a client connects to a producer with newer mutations that are not present on the producer. The client must rollback or undo some mutations to align with the mutations on the producer. If the producer is rolled back to 0, it may be undesirable because it means the service is attempting to resynchronize the entirety of the data set.

Reply:
I'd perhaps replace "producer" with "source" (or even "the Data Service") but otherwise that looks great!

(Don't worry about length in the docs - better that they're comprehensive than concise.)

Reply:
I'll make that change now then thank you :)